imgui-render-opengl3/imgui/buildfile

```diff
@@ -0,0 +1,38 @@
+intf_libs = # Interface dependencies.
+import intf_libs += opengl-meta%lib{opengl-gl}
+import intf_libs += imgui%lib{imgui}
+impl_libs = # Implementation dependencies.
+
+lib{imgui-render-opengl3}: {hxx cxx}{**} $impl_libs $intf_libs
+
+# Build options.
+out_pfx_inc = [dir_path] $out_root/imgui/
+src_pfx_inc = [dir_path] $src_root/imgui/
+
+cxx.poptions =+ "-I$out_pfx_inc" "-I$src_pfx_inc" -DIMGUI_EXPORT
```

Comment:
I believe this is wrong: defining `IMGUI_EXPORT` while building the backend will case core headers to export instead of importing symbols (I wonder how does it even build on Windows).

I took a look and for backends it uses `IMGUI_IMPL_API` instead of `IMGUI_API`. So I think you will need to invent parallel `IMGUI_IMPL_EXPORT` and define `IMGUI_IMPL_API` based on that in `imconfig.h`, similar to how you do it for `IMGUI_API`.

Note: there are the backend with the same issue.

Reply:
Thanks, changed to a impl version for backends.